test_help_preview_generation fails with rich==13.8.1

Hi!

We are using rich-argparse 1.5.2 and after upgrading rich from 13.7.1 to 13.8.1 in nixpkgs the following test started failing:

```pytb
_________________________ test_help_preview_generation _________________________

tmp_path = PosixPath('/build/pytest-of-nixbld/pytest-0/test_help_preview_generation0')

    def test_help_preview_generation(tmp_path):
        parser = ArgumentParser("PROG", formatter_class=RichHelpFormatter)
        parser.add_argument("--foo", help="foo help")
        preview_action = parser.add_argument("--generate", action=HelpPreviewAction)
        default_path = tmp_path / "default-preview.svg"
        parser.add_argument("--generate-with-default", action=HelpPreviewAction, path=str(default_path))
    
        # No namespace pollution
        args = parser.parse_args(["--foo", "FOO"])
        assert vars(args) == {"foo": "FOO"}
    
        # No help pollution
        assert "--generate" not in parser.format_help()
    
        # No file, error
        with pytest.raises(SystemExit) as exc_info:
            parser.parse_args(["--generate"])
        assert exc_info.value.code == 1
    
        # Default file, ok
        with pytest.raises(SystemExit) as exc_info:
            parser.parse_args(["--generate-with-default"])
        assert exc_info.value.code == 0
        assert default_path.exists()
    
        # SVG file
        svg_file = tmp_path / "preview.svg"
        with pytest.raises(SystemExit) as exc_info:
            parser.parse_args(["--generate", str(svg_file)])
        assert exc_info.value.code == 0
        assert svg_file.exists()
        assert svg_file.read_text().startswith("<svg")
    
        # HTML file
        preview_action.export_kwds = {}
        html_file = tmp_path / "preview.html"
        with pytest.raises(SystemExit) as exc_info:
            parser.parse_args(["--generate", str(html_file)])
        assert exc_info.value.code == 0
        assert html_file.exists()
        assert html_file.read_text().startswith("<!DOCTYPE html>")
    
        # TXT file
        preview_action.export_kwds = {}
        txt_file = tmp_path / "preview.txt"
        with pytest.raises(SystemExit) as exc_info:
            parser.parse_args(["--generate", str(txt_file)])
        assert exc_info.value.code == 0
        assert txt_file.exists()
>       assert txt_file.read_text().startswith("Usage:")
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x7ffff7f51c90>('Usage:')
E        +    where <built-in method startswith of str object at 0x7ffff7f51c90> = ''.startswith
E        +      where '' = read_text()
E        +        where read_text = PosixPath('/build/pytest-of-nixbld/pytest-0/test_help_preview_generation0/preview.txt').read_text

tests/test_argparse.py:992: AssertionError
----------------------------- Captured stderr call -----------------------------
error: help preview path is not provided
Help preview saved to /build/pytest-of-nixbld/pytest-0/test_help_preview_generation0/default-preview.svg
Help preview saved to /build/pytest-of-nixbld/pytest-0/test_help_preview_generation0/preview.svg
Help preview saved to /build/pytest-of-nixbld/pytest-0/test_help_preview_generation0/preview.html
Help preview saved to /build/pytest-of-nixbld/pytest-0/test_help_preview_generation0/preview.txt
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,9 @@
 ## Unreleased
 
 ### Fixes
+- [GH-133](https://github.com/hamdanal/rich-argparse/issues/133),
+  [PR-135](https://github.com/hamdanal/rich-argparse/pull/135)
+  Fix help preview generation with newer releases of rich.
 - [GH-130](https://github.com/hamdanal/rich-argparse/issues/130),
   [PR-131](https://github.com/hamdanal/rich-argparse/pull/131)
   Fix a bug that caused long group titles to wrap.

--- a/rich_argparse/__init__.py
+++ b/rich_argparse/__init__.py
@@ -558,11 +558,11 @@ class HelpPreviewAction(argparse.Action):
             parser.exit(1, "error: help preview path must be a string\n")
         if not path.endswith((".svg", ".html", ".txt")):
             parser.exit(1, "error: help preview path must end with .svg, .html, or .txt\n")
+        import io
 
         text = r.Text.from_ansi(parser.format_help())
-        console = r.Console(record=True)
-        with console.capture():
-            console.print(text, crop=False)
+        console = r.Console(file=io.StringIO(), record=True)
+        console.print(text, crop=False)
 
         if path.endswith(".svg"):
             self.export_kwds.setdefault("title", "")

--- a/tests/test_argparse.py
+++ b/tests/test_argparse.py
@@ -978,7 +978,9 @@ def test_help_preview_generation(tmp_path):
         parser.parse_args(["--generate", str(svg_file)])
     assert exc_info.value.code == 0
     assert svg_file.exists()
-    assert svg_file.read_text().startswith("<svg")
+    svg_out = svg_file.read_text()
+    assert svg_out.startswith("<svg")
+    assert "Usage" in svg_out
 
     # HTML file
     preview_action.export_kwds = {}
@@ -987,7 +989,9 @@ def test_help_preview_generation(tmp_path):
         parser.parse_args(["--generate", str(html_file)])
     assert exc_info.value.code == 0
     assert html_file.exists()
-    assert html_file.read_text().startswith("<!DOCTYPE html>")
+    html_out = html_file.read_text()
+    assert html_out.startswith("<!DOCTYPE html>")
+    assert "Usage" in html_out
 
     # TXT file
     preview_action.export_kwds = {}

